rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateJavaCompatibility.java

```diff
@@ -63,6 +64,21 @@ public class UpdateJavaCompatibility extends Recipe {
     @Nullable
     Boolean allowDowngrade;
 
+    @Option(displayName = "Add Source Compatibility if missing",
+            description = "Adds Source Compatibility if one is not found.",
+            required = false)
+    @Nullable
+    Boolean addSourceCompatibilityIfMissing;
+
+    @Option(displayName = "Add Target Compatibility if missing",
+            description = "Adds Target Compatibility if one is not found.",
+            required = false)
+    @Nullable
+    Boolean addTargetCompatibilityIfMissing;
```

Comment:
Given the option that allows picking which type to update -- both, source only, or target only -- this could probably be simplified to just a single `addIfMissing`.

Reply:
I'm fine with that.  This way was a little more flexible but there's probably not very many cases where you would want one option and not the other as well.

Reply:
Could we accomplish both simplifying this and maintaining flexibility by changing the `Boolean` to an `Enum` with values of `SourceOnly`, `TargetOnly`, `SourceAndTarget`?